docs/conceptual/TOC.md

```diff
@@ -79,6 +79,7 @@
 ####[Flexible Types](flexible-types-[fsharp].md)
 ####[Delegates](delegates-[fsharp].md)
 ####[Object Expressions](object-expressions-[fsharp].md)
+####[Copy and Update Record Expressions](copy-and-update-record-expression-[fsharp].md)
```

Comment:
Nit - can you rename to `Copy and Update Record Expressions`, so it's in its plural form, just like other docs that cover expressions?